src/app/query/global-query/global-query.component.html

```diff
@@ -0,0 +1,120 @@
+<div class="container-main">
+    <div class="query-container py-4">
+        <div class="box p-4 schemes-container">
```

Comment:
Nitpick: Please rename `Dataset shemes` to `Dataset schemas`. 

"Schema" is what defines data structure, while "scheme" usually describes a process (e.g. `http://` vs `https://` is a protocol scheme).

Reply:
Fixed